notebooks/tutorials/4_futures_getting_started_lesson2/notebook.ipynb

```diff
@@ -63,7 +63,7 @@
     {
      "data": {
       "text/plain": [
-       "Future(1058201601, symbol=u'CLF16', root_symbol=u'CL', asset_name=u'Light Sweet Crude Oil', exchange=u'NYMEX', start_date=Timestamp('2012-01-19 00:00:00+0000', tz='UTC'), end_date=Timestamp('2015-12-21 00:00:00+0000', tz='UTC'), first_traded=None, notice_date=Timestamp('2015-12-23 00:00:00+0000', tz='UTC'), expiration_date=Timestamp('2015-12-21 00:00:00+0000', tz='UTC'), auto_close_date=Timestamp('2015-12-17 00:00:00+0000', tz='UTC'), tick_size=0.01, multiplier=1000.0, exchange_full=u'NYMEX')"
+       "Future(1058201601 [CLF16])"
```

Comment:
Do we no longer include all the other information as part of the futures object?

Reply:
No, we updated the representations of  continuous futures and contracts.

Reply:
So this is just an update to the representation? Can I still get the `root_symbol` or the `exchange` or `expiration_date`?

Reply:
correct.